src/main/java/com/thealgorithms/others/EulersFunction.java

```diff
@@ -1,17 +1,27 @@
 package com.thealgorithms.others;
 
 /**
- * You can read more about Euler's totient function
- *
- * <p>
- * See https://en.wikipedia.org/wiki/Euler%27s_totient_function
+ * @brief utility class for <a href="https://en.wikipedia.org/wiki/Euler%27s_totient_function">Euler's totient function</a>
  */
-public class EulersFunction {
+final public class EulersFunction {
+    private EulersFunction() {
+    }
```

Comment:
Why we are restricted from making an object of this class?

Reply:
The class `EulersFunction` has no non-`static` members. Therefore, in my opinion, having objects of that class does not make too much sense. That is why I decided to make the default constructor `private`.

Also thanks to this, `jacoco` shows _100%_ code coverage (because it is not complaining about not creating objects of the class `EulersFunction`).